### PR TITLE
[*] Models - Prevent bad Sequelize model generation if origin table does not contain `id` column nor primary keys.

### PR DESCRIPTION
**Pull request by larcin**
Doesn't fix that much stuff. But now on tables without column id and primary keys, the error doesn't come form the generated models anymore.

### CODE REVIEW

**Comment by larcin** [comment]
[Lumber - Should add removeAttribute 'id' when not present](https://trello.com/c/OJK6iPCr/3588-lumber-should-add-removeattribute-id-when-not-present)

**Comment by larcin** [comment]
[Lumber - Should add removeAttribute 'id' when not present](https://trello.com/c/OJK6iPCr/3588-lumber-should-add-removeattribute-id-when-not-present)

File: utils/table-helper.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+// NOTICE: Look for the id column in both fields and primary keys
+function hasIdColumn(fields, primaryKeys) {
+  return fields.some(field => field.name === 'id' || field.nameColumn === 'id') ||
+        _.includes(primaryKeys, 'id');
+}
+
+module.exports = {
+  hasIdColumn,
+};
```
Comment (arnaudForestAdmin) [inline utils/table-helper.js]: I think that moving this logic in the `database-analyzer.js` file and passing the `hasIdColumn` info in the `options` result of `analyzeTable` function would really reduce the code size.

File: utils/table-helper.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const _ = require('lodash');
+
+// NOTICE: Look for the id column in both fields and primary keys
+function hasIdColumn(fields, primaryKeys) {
+  return fields.some(field => field.name === 'id' || field.nameColumn === 'id') ||
+        _.includes(primaryKeys, 'id');
```
Comment (arnaudForestAdmin) [inline utils/table-helper.js]: [Code Style] Maybe we should add ESLint, the indentation is pretty bad here.
To replace by:

```javascript
return fields.some(field => field.name === 'id' || field.nameColumn === 'id')
  || _.includes(primaryKeys, 'id');
```

File: services/dumper.js [hunk 3]
```diff
@@ -218,8 +221,10 @@ function Dumper(config) {
     copyTemplate('middlewares/welcome/template.txt', `${middlewaresPath}/welcome/template.txt`);
   }
 
-  this.dump = (table, { fields, references, options }) => {
-    writeModel(path, table, fields, references, options);
+  this.dump = (table, {
+    fields, references, primaryKeys, options,
```
Comment (arnaudForestAdmin) [inline services/dumper.js]: [Code Style] Not sure this is conventional for the Airbnb style.
```javascript
{
  fields,
  references,
  primaryKeys,
  options,
}
```
?

File: services/database-analyzer.js [hunk 4]
```diff
@@ -49,6 +49,12 @@ function DatabaseAnalyzer(databaseConnection, config) {
     return alias;
   }
 
+  // NOTICE: Look for the id column in both fields and primary keys
+  function hasIdColumn(fields, primaryKeys) {
+    return fields.some(field => field.name === 'id' || field.nameColumn === 'id')
+      || _.includes(primaryKeys, 'id');
```
Comment (arnaudbesnier) [inline services/database-analyzer.js]: Not sure this is the "perfect" criteria to set the `removeAttribute('id')` in the model.

**Pull request merged**